pkg/sql/parsers/dialect/mysql/mysql_sql_test.go

```diff
@@ -1389,6 +1389,12 @@ var (
 		}, {
 			input:  "create index idx1 using btree on A (a) KEY_BLOCK_SIZE 10 with parser x comment 'x' invisible",
 			output: "create index idx1 using btree on a (a) KEY_BLOCK_SIZE 10 with parser x comment x invisible",
+		}, {
+			input:  "create index idx using ivfflat on A (a) LISTS 10",
```

Comment:
CREATE INDEX <idx_name> USING ivfflat  ON <table_name> LIST 10;
Are there any instructions for using this grammar？

Reply:
Yes.

An example is
```sql
create index idx using IVFFLAT on t1(name) lists = 10 similarity_function 'l2';
```

Reply:
If you choose `LIST 10` alone , and miss out `similarity function = 'ip' `, we pick the default similarity function ie `'cos'`.

Reply:
> CREATE INDEX <idx_name> USING ivfflat ON <table_name> LIST 10;

Right now, the IVFFLAT index is not yet implemented. It will be implemented in Part 3 PR.